interfaces/AW-App/src/app/validation-components/scan-qr/scan-qr.component.ts

```diff
@@ -57,9 +57,15 @@ export class ScanQrComponent implements ValidationComponent {
   "programId": 1 }`,
           }
         : {};
+    const cssClass =
+      environment.isDebug || environment.showDebug
+        ? ''
```

Comment:
`cssClass` will always be an empty string now. I think you want to add `'debugModal'` to the first `''`.